cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/wizard-basic/UnmanagedClusterBasic.tsx

```diff
@@ -0,0 +1,24 @@
+// React imports
```

Comment:
It is probably best to just remove this file from this pull request since it has a lot of references to AWS components and data store. It will make this PR cleaner but more importantly will make your commit to the main repo cleaner if you ever have to go back to this specific commit.